apps/src/aichat/types.ts

```diff
@@ -65,6 +65,10 @@ export interface LogChatEventApiResponse {
   chat_event: ChatMessage;
 }
 
+export interface UserHasAichatAccessResponse {
```

Comment:
Personal preference: since this type isn't shared and only referenced internally inside a function in `aichatApi` might be easier to just have this alongside the function? But fine either way

Reply:
I don't have a preference so don't mind moving it.